packages/playground/build-scripts-storybook/samples-prepare.js

```diff
@@ -73,6 +75,18 @@ export type StoryArgsSlots = {
 						control: "select",
 						options: typeEnum.properties.map(a => a.type),
 					};
+				} else if (NUMERIC_TYPES.indexOf(typeEnum?.name) > -1) {
```

Comment:
We can use `NUMERIC_TYPES.includes(typeEnum?.name)`. Same for next expression.

Reply:
Good point!
Done